workflow/models.py

```diff
@@ -125,6 +147,11 @@ class Organization(models.Model):
     chargebee_subscription_id = models.CharField(blank=True, null=True, max_length=50)
     chargebee_used_seats = models.IntegerField(blank=True, null=True, default=0)
     oauth_domains = fields.ArrayField(models.CharField("OAuth Domains", max_length=255, null=True, blank=True), null=True, blank=True)
+    date_format = models.CharField("Date Format", max_length=50, blank=True, default="DD.MM.YYYY")
+    default_currency = models.ForeignKey(Currency, blank=True, null=True)
+    currency_format = models.CharField("Currency Format", max_length=50, blank=True, default="Commas")
```

Comment:
Why the default is Commas?

Reply:
That simply means something like 120,000,000. Which has been the system default.

Reply:
@jefmoura : after a talk with Greg, we decided to remove the language field, and move the feature to next release at least untill the internalization endpoint is ready and being consumed.I have removed the field and committed. Kindly merge the branch to unblock the frontend tickets